cfep-18.md

```diff
@@ -0,0 +1,79 @@
+
+<table>
+<tr><td> Title </td><td> Packaging static libraries </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Uwe L. Korn &lt;cfep@uwekorn.com&gt;</td></tr>
+<tr><td> Created </td><td> Jun 24, 2020</td></tr>
+<tr><td> Updated </td><td> Jun 24, 2020</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CFEP proposes a policy on how to package static libraries / archives.
+These should be separated out into packages that are not installed in a default user setup.
+In general, we prefer our packages to work completely without static linkage between different packages but are aware that static archives are sometimes needed / useful.
+
+This CFEP excludes languages like Rust, Haskell, Go where binaries are always statically linked.
+The mentioned downsides apply there too but there isn't the option of using dynamic linkage.
+
+## Motivation
+
+At the moment, most binary packages that include libraries only ship their shared version.
+Some packages though also include the static version or only ship the static version.
+At runtime the static archives are not needed and thus are wasted space to the end user.
+In addition, our tooling (conda/conda-build/the bots) is also geared towards maintaining a set of dynamically linked packages.
+
+## Rationale
+
+The current stack used to manage and consume conda-forge is tuned to deal with static linkages between packages.
```

Comment:
Is `static linkages` correct here?  I though that conda-forge was tuned to deal with dynamic linking.

Reply:
No, it isn't. It should be 
```suggestion
The current stack used to manage and consume conda-forge is tuned to deal with dynamic linkage between packages.
```